src/resources/parser/glb-parser.js

```diff
@@ -796,10 +796,10 @@ var createMesh = function (device, gltfMesh, accessors, bufferViews, callback, d
                     options.name = targets.length.toString(10);
                 }
 
-                targets.push(new MorphTarget(device, options));
+                targets.push(new MorphTarget(options));
             });
 
-            mesh.morph = new Morph(targets);
+            mesh.morph = new Morph(targets, device);
```

Comment:
Maybe I mentioned this at the time, but just for the record, the rest of the engine API _always_ takes the device as the first parameter if it's required. So this is non-conforming. But approving anyway.